### ISSUE TITLE
Can't request to endpoint with dynamic PathTemplate

### ISSUE THREAD

**Issue by MinhMit** [ThreeMammals/Ocelot#1020]
## Expected Behavior / New Feature
Need request to endpoint when use dynamic {everythings}

## Actual Behavior / Motivation for New Feature
I have a service with url:

http://logging.mydomain.com/api/logs
http://logging.mydomain.com/api/logs/trace
http://logging.mydomain.com/api/logs/write
And a api gateway with url:
http://api.mydomain.com

in ocelot configuration file:
```
{
  "ReRoutes": [
    {
      "DownstreamPathTemplate": "/api/logs/{everything}",
      "DownstreamScheme": "http",
      "DownstreamHostAndPorts": [
        {
          "Host": "logging.mydomain.com",
          "Port": 80
        }
      ],
      "UpstreamPathTemplate": "/logs/{everything}",
      "UpstreamHttpMethod": [ "Get", "Post", "Put", "Delete", "Options" ]
    }
  ],
  "GlobalConfiguration": {
    "BaseUrl": "http://api.mydomain.com"
  }
}
```
I just can redirect to 
http://logging.mydomain.com/api/logs/trace
http://logging.mydomain.com/api/logs/write
and can't redirect to 
http://logging.mydomain.com/api/logs

Thanks for your support

## Steps to Reproduce the Problem

  1.
  1.
  1.

## Specifications

  - Version:
  - Platform: .net core 2.2
  - Subsystem:

**Comment by raman-m** [comment]
> `"DownstreamPathTemplate": "/api/logs/{everything}",`
and can't redirect to http://logging.mydomain.com/api/logs

This issue was fixed by PR #1911 and released in version [23.0](https://github.com/ThreeMammals/Ocelot/releases/tag/23.0.0)
See: [Empty Placeholders](https://ocelot.readthedocs.io/en/latest/features/routing.html#empty-placeholders) feature
Please use the latest versions 23.0+ :exclamation:

**Comment by raman-m** [comment]
Fixed by #1911 
- #1911
